packages/wallets-helpers/rollup.config.js

```diff
@@ -0,0 +1,50 @@
+import * as process from 'node:process';
+import fs from 'node:fs';
+import ts from 'typescript';
+import del from 'rollup-plugin-delete';
+import resolve from '@rollup/plugin-node-resolve';
+import typescript from 'rollup-plugin-typescript2';
+import { babel } from '@rollup/plugin-babel';
+import json from '@rollup/plugin-json';
+
+const extensions = ['.js', '.jsx', '.ts', '.tsx'];
+const { dependencies = {}, peerDependencies = {} } =
+  JSON.parse(fs.readFileSync('package.json', 'utf-8'));
+const commonExternal = [
+  'react/jsx-runtime',
+  // Do not include in the bundle subpath exports like:
+  /^@reef-knot\/.*/, // e.g. @reef-knot/<package>/<exports-field-entry>
+  /^reef-knot\/.*/, // e.g. reef-knot/wallets-icons/react
+];
+const external = [
+  ...commonExternal,
+  ...Object.keys({ ...dependencies, ...peerDependencies }),
+  /node_modules/
+];
+const isDevMode = process.env.dev === 'on';
```

Comment:
usually env variables are capitalised, this feels a bit weird.
Usually `NODE_ENV` is used for this purposes

Reply:
There is such script in package.json:
```
"dev": "dev=on rollup -c -w",
```
I guess it can be improved later.